test/src/test/java/org/corfudb/runtime/CheckpointLivenessUpdaterUnitTest.java

```diff
@@ -1,8 +1,7 @@
-package org.corfudb.runtime.checkpoint;
+package org.corfudb.runtime;
```

Comment:
There are other tests that require this change. That can go as a separate cleanup CL.